sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -0,0 +1,21 @@
+## API Report File for "@azure/tables"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export interface Foo {
```

Comment:
where does this interface come from? I don't see it inside index.ts?

Reply:
This was just from the default template. It's updated now